citi-onboarding/colisio
# Criacao do CRUD de processos

## What I did

- CRUD criado para gerenciamento de informações do serviço prestado.



> - Adição do método criar, ler, atualizar e deletar para processos



### How to test
1. Open the terminal and go to the folder of this project


2. In terminal:


    git checkout develop
    git pull
    git checkout
    git pull origin
    yarn install
    yarn start

3. In your browser go to http://localhost:port/

4. Check if the component looks just like the mockup in different screen sizes

5. Read the changed files section on GitHub

6. Test the things that you think are worth testing, even the ones that are not in this description
